src/sentry/release_health/metrics_sessions_v2.py

```diff
@@ -364,6 +364,8 @@ class SimpleForwardingField(Field):
         "crash_rate(user)": SessionMRI.CRASH_USER_RATE,
         "crash_free_rate(session)": SessionMRI.CRASH_FREE_RATE,
         "crash_free_rate(user)": SessionMRI.CRASH_FREE_USER_RATE,
+        "anr_rate()": SessionMRI.ANR_RATE,
+        "foreground_anr_rate()": SessionMRI.FOREGROUND_ANR_RATE,
```

Comment:
Is there any reason for not putting `user` between `()`?

Reply:
No reason other than that ANR rate by definition only accounts for unique users and it's not really parametrizable to take in any other input other than users

Reply:
Great! I thought so by looking at the derived metrics.